### DISCUSSION TITLE
[@types/chrome] chrome.storage.sync.get()  type is error

### DISCUSSION THREAD

**Discussion by leezhian** [DefinitelyTyped/DefinitelyTyped#63154] [Issues with a @types package]
for example：
```javascript
const res = await chrome.storage.sync.get(null)

console.log(res.a) // error, Property 'a' does not exist on type 'void'.
```

but This only returns a Promise when the callback parameter is not specified, and with MV3+. The type inside the Promise is the same as the 1st argument to callback.

[a comment by an automated account is not shown]

**Comment by nicolas377** [comment]
Could you make a reproduction in the [playground](https://www.typescriptlang.org/play)? I can't seem to reproduce the error with the reproduction you gave.
